src/main/webapp/script.js

```diff
@@ -11,3 +11,81 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
+var jsonPlacehold = '{ "questions" : [' +
+'{ "q":"How would you spend an afternoon?" , "options":["hiking mountains", "shopping crafts", "trying local eats", "visiting a museum"]},' +
+'{ "q":"What would you choose for breakfast?" , "options":["acai bowl", "waffles", "crossaints", "huevos rancheros", "bagels"] },' +
```

Comment:
When this gets sent across the wire, it's fairly cheap since we're only sending it once. I'd recommend the full word "question" instead of q.